sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -4,6 +4,20 @@ var srlog = SIREPO.srlog;
 var srdbg = SIREPO.srdbg;
 
 SIREPO.NUMBER_REGEXP = /^\s*(\-|\+)?(\d+|(\d*(\.\d*)))([eE][+-]?\d+)?\s*$/;
+SIREPO.SAFE_PATH_REGEXP_EXCLUDE_CHARS = [   // array of illegal filename characters
+    '\\\\',  // doubly escaped
```

Comment:
Side comment is unnecessary.

I would avoid too much escaping. It would be clearer to do this, and I'll demonstrate usage later:

```javascript
SIREPO.UNSAFE_PATH_CHARS = '\\/&:+?\'"<>|'.split('');
SIREPO.UNSAFE_PATH_WARN = must not include any of the following: '
    + SIREPO.UNSAFE_PATH_CHARS.join(' ');
SIREPO.UNSAFE_PATH_RE = new.RegExp('[\\' + SIREPO.UNSAFE_PATH_CHARS.join('\\') + ']');
```